File: Richest Customer Wealth/richestcustomer.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    var max;
    for(let i = 0; i < accounts.length; i++){
        var curr = 0;
        for(let j = 0; j < accounts[i].length; j++){
            curr += accounts[i][j];
        }
        if(max < curr || max == null){
            max = curr;
        }
    }
    return max;
};

File: Richest Customer Wealth/richest_customer.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max = 0
        for i in range(len(accounts)):
            curr = 0
            for j in range(len(accounts[i])):
                curr += accounts[i][j]
            if(curr > max):
                max = curr
        return max
    